Python/Tests/Core/DebugReplEvaluatorTests.cs

```diff
@@ -63,6 +65,7 @@ public class DebugReplEvaluatorTests {
             _window = new MockReplWindow(_evaluator);
             _evaluator._Initialize(_window);
             _processes = new List<PythonProcess>();
+            _moduleFileExtension = Version.Version < Microsoft.PythonTools.Parsing.PythonLanguageVersion.V30 && Version.IsCPython ? ".pyc" : ".py";
```

Comment:
Another alternative - when comparing paths use `Path.ChangeExtension(path, null)` on both to remove the extension first.